pkg/activator/revision.go

```diff
@@ -143,17 +141,17 @@ func (r *revisionActivator) getRevisionEndpoint(revision *v1alpha1.Revision) (en
 func (r *revisionActivator) ActiveEndpoint(namespace, name string) ActivationResult {
 	key := fmt.Sprintf("%s/%s", namespace, name)
 	logger := r.logger.With(zap.String(logkey.Key, key))
-	revision, err := r.activateRevision(namespace, name)
+	revision, err := r.activateRevision(namespace, name, key)
 	if err != nil {
-		logger.Errorw("Failed to activate the revision.", zap.Error(err))
+		logger.Errorw("failed to activate the revision", zap.Error(err))
```

Comment:
This is a log message and not an error. The capitalization looked fine.

Reply:
Got carried away, I guess.